docs/concepts/services-networking/service.md

```diff
@@ -124,8 +124,8 @@ Accessing a `Service` without a selector works the same as if it had a selector.
 The traffic will be routed to endpoints defined by the user (`1.2.3.4:9376` in
 this example).
 
-An ExternalName service is a special case of service that does not have
-selectors. It does not define any ports or Endpoints. Rather, it serves as a
+An `ExternalName` service is a special case of service that does not have
+selectors. It does not define any ports or `Endpoints`. Rather, it serves as a
 way to return an alias to an external service residing outside the cluster.
 
```

Comment:
@paul-rogers Thanks for doing this. We use caps for Kubernetes objects, but we don't use backticks. So could you go through and change `Endpoints` to Endpoints, `Service` to Service, `Pod` to Pod, etc. Thanks